nilearn/plotting/img_plotting.py

```diff
@@ -366,34 +366,39 @@ def plot_anat(anat_img=MNI152TEMPLATE, cut_coords=None,
         black_bg: boolean, optional
             If True, the background of the image is set to be black. If
             you wish to save figures with a black background, you
-            will need to pass "facecolor='k', edgecolor='k'" to pylab's
-            savefig.
+            will need to pass "facecolor='k', edgecolor='k'" to plt.savefig.
         cmap: matplotlib colormap, optional
             The colormap for the anat
+        vmin: float
+            vmin passed to plt.imshow
```

Comment:
lower/upper bound is probably a better phrasing.

Analogy to np.clip is probably not usable for our non-advanced users. The advanced one should understand vmin, vmax as it is standard.